pkg/storaged/btrfs/btrfs-tool.py

```diff
@@ -16,13 +16,15 @@
 # wont get confused.
 
 import contextlib
+import ctypes
```

Comment:
Please add "Fixes #21355" and https://issues.redhat.com/browse/COCKPIT-1210 to the commit message in the next round.

Reply:
Done.